src/main/java/org/fcrepo/importexport/importer/Importer.java

```diff
@@ -648,26 +651,27 @@ public static boolean verifyBag(final File bagDir) {
     /**
      * Method to find and set the repository root from the resource uri.
      * @param uri the URI for the resource
-     * @throws IOException
-     * @throws FcrepoOperationFailedException
      */
-    private void findRepositoryRoot(final URI uri) {
-        repositoryRoot = uri;
+    @VisibleForTesting
+    public URI findRepositoryRoot(final URI uri) {
```

Comment:
If we keep the one integration test, this will need to continue to be public.  Since this is primarily a command-line utility, and the method doesn't impact the internal state of the importer instance, I think this is fine.